src/OrbitQt/orbitglwidget.cpp

```diff
@@ -34,6 +34,9 @@
 #define ORBIT_DEBUG_OPEN_GL 0
 
 OrbitGLWidget::OrbitGLWidget(QWidget* parent) : QOpenGLWidget(parent) {
+  QSurfaceFormat requested_format = QSurfaceFormat::defaultFormat();
+  LOG("OpenGL major version requested: %i", requested_format.majorVersion());
+  LOG("OpenGL minor version requested: %i", requested_format.minorVersion());
```

Comment:
Nit: I guess this could be a single log line outputting both major and minor version number.

Reply:
Agree, changed.